pandas/tests/frame/test_timeseries.py

```diff
@@ -120,6 +122,36 @@ def test_pct_change_shift_over_nas(self):
         edf = DataFrame({'a': expected, 'b': expected})
         assert_frame_equal(chg, edf)
 
+    def test_pct_change_periods_freq(self):
+        # see issue #7292
+        rs_freq = self.tsframe.pct_change(freq='5B')
+        rs_periods = self.tsframe.pct_change(5)
+        assert_frame_equal(rs_freq, rs_periods)
+
+        rs_freq = self.tsframe.pct_change(freq='3B', fill_method=None)
+        rs_periods = self.tsframe.pct_change(3, fill_method=None)
+        assert_frame_equal(rs_freq, rs_periods)
+
+        rs_freq = self.tsframe.pct_change(freq='3B', fill_method='bfill')
+        rs_periods = self.tsframe.pct_change(3, fill_method='bfill')
+        assert_frame_equal(rs_freq, rs_periods)
+
+        rs_freq = \
```

Comment:
We try to avoid `\` for line continuation in pandas. We use parenthesis instead. (same comment for lines 144 and 149)

Reply:
make sense :) thanks